org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/validation/AssertableDiagnostics.java

```diff
@@ -20,6 +20,7 @@
 import java.util.Map.Entry;
 
 import org.eclipse.emf.common.util.Diagnostic;
```

Comment:
@szarnekow  should we add a `@since 2.26` to class javadoc?

Reply:
Yes, please.

Reply:
done